Sources/Opus/Opus.Decoder.swift

```diff
@@ -22,6 +23,27 @@ public extension Opus {
 			}
 		}
 
+		public init(customOpus: OpaquePointer,
+		            format: AVAudioFormat,
+		            frameSize: UInt32) throws
```

Comment:
This should be named `customFrameSize` to be consistent with the internal representation and Opus documentation.